### ISSUE TITLE
Oh No! Donations failing to DAC in DAI and PAN

### ISSUE THREAD

**Issue by Danibelle** [Giveth/giveth-dapp#1534]
We tried Infinite Unlock and Regular Unlock & Donate with DAI..

Then regular Unlock and Donate with PAN.

Sign first transaction, receive notification to wait for second transaction.. it never comes then error message:

![image](https://user-images.githubusercontent.com/39466970/99840351-00665e00-2b32-11eb-80e5-db919441f6da.png)

**Comment by GriffGreen** [comment]
Dani to reproduce

**Comment by GriffGreen** [comment]
Attach one of those pictures of the console

**Comment by Danibelle** [comment]
This is likely the same OH NO! message that keeps pooping up but the transaction still succeeds.

**Comment by aminlatifi** [comment]
> This is likely the same OH NO! message that keeps pooping up but the transaction still succeeds.

@Danibelle We pushed an update around Jan 13, did you encounter such error after that?

**Comment by GriffGreen** [comment]
Can be tested on develop...

**Comment by laurenluz** [comment]
Hey @aminlatifi I'm not sure I can do this... is it possible to even get Ropsten DAI or PAN?

**Comment by aminlatifi** [comment]
@laurenluz We have a test token on develop, I can send you some amount. What is your wallet address?

**Comment by laurenluz** [comment]
@aminlatifi 0xC46c67Bb7E84490D7EbdD0b8ecDaca68Cf3823F4

Thanks!

**Comment by aminlatifi** [comment]
@laurenluz Sent, it's token contract address to add to your metamask 0xe690e380740a682e2b8ceaea33584ea2cb59849e

**Comment by laurenluz** [comment]
This issue has been tested and resolved (found no issues donating to DACs w the Test Token), details below:

Tried regular unlock, donating to a DAC w Test Token, signed first transaction (all good), signed second - all good!

Then tried infinite unlock, donating to a Campaign w Test Token, signed first transaction (all good)... had to wait a minute for the second transaction, but it came up, I signed and everything was good!

Then I tried donating to "Giveth" or in this case, "hola, mundo" - no issues.

Then I tried donating to another DAC - no issues.

![image](https://user-images.githubusercontent.com/75490971/106367347-99baa900-6307-11eb-92b6-9d60cc5ddbc9.png)

![image](https://user-images.githubusercontent.com/75490971/106367816-04b9af00-630b-11eb-85bf-34d7cae89389.png)

![image](https://user-images.githubusercontent.com/75490971/106368084-a261ae00-630c-11eb-90ef-23658594a6b0.png)

![image](https://user-images.githubusercontent.com/75490971/106368102-d046f280-630c-11eb-9324-d19e6d1ceabf.png)
